pkg/aws/pca.go

```diff
@@ -99,7 +99,7 @@ func (p *PCAProvisioner) Sign(ctx context.Context, cr *cmapi.CertificateRequest,
 
 	validityExpiration := int64(time.Now().Unix()) + 30*24*3600
```

Comment:
We may want to set the default expiry here to 90 days as suggested by cert-manager https://cert-manager.io/docs/usage/certificate/#renewal. LMK and i'll update that as well